2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_setup.py:_flush():76] Configure stats pid to 14936
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_b', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': '7498vb74', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-w52d5302/config-7498vb74.yaml', 'sweep_id': 'w52d5302'}
2022-08-01 18:58:35,819 WARNING Thread-21 :14936 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_185835-7498vb74/logs/debug.log
2022-08-01 18:58:35,819 INFO    Thread-21 :14936 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_185835-7498vb74/logs/debug-internal.log
2022-08-01 18:58:35,821 INFO    Thread-21 :14936 [wandb_init.py:init():495] calling init triggers
2022-08-01 18:58:35,821 INFO    Thread-21 :14936 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0005352289278704362, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
config: {}
2022-08-01 18:58:35,822 INFO    Thread-21 :14936 [wandb_init.py:init():548] starting backend
2022-08-01 18:58:35,822 INFO    Thread-21 :14936 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 18:58:35,827 INFO    Thread-21 :14936 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 18:58:35,844 INFO    Thread-21 :14936 [backend.py:ensure_launched():222] started backend process with pid: 24613
2022-08-01 18:58:35,845 INFO    Thread-21 :14936 [wandb_init.py:init():558] backend started and connected
2022-08-01 18:58:35,859 INFO    Thread-21 :14936 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0005352289278704362, 'model': 'COCO-Detection/faster_rcnn_R_101_DC5_3x.yaml'}
2022-08-01 18:58:35,864 INFO    Thread-21 :14936 [wandb_init.py:init():634] updated telemetry
2022-08-01 18:58:35,865 INFO    Thread-21 :14936 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 18:58:36,647 INFO    Thread-21 :14936 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 18:58:36,686 INFO    Thread-21 :14936 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 18:58:36,686 INFO    Thread-21 :14936 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 18:58:37,887 INFO    Thread-21 :14936 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 18:58:37,890 INFO    Thread-21 :14936 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 18:58:37,891 INFO    Thread-21 :14936 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 18:58:37,891 ERROR   Thread-21 :14936 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 18:58:37,892 INFO    Thread-21 :14936 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 19:14:15,152 INFO    MainThread:14936 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2022-08-01 19:14:15,161 INFO    MainThread:14936 [wandb_run.py:_restore():1834] restore
